qcelemental/molparse/to_string.py

```diff
@@ -143,6 +144,31 @@ def to_string(molrec: Dict,
             'coordinates': 'cartesian',
         }
 
+    elif dtype == 'molpro':
+
+        atom_format = '{elem}'
+        ghost_format = '{elem}'
+        umap = {'bohr': 'bohr', 'angstrom': 'angstrom'}
+
+        atoms = _atoms_formatter(molrec, geom, atom_format, ghost_format, width, prec, 2)
+
+        ghost_atoms = []
+        nat = geom.shape[0]
+        for iat in range(nat):
+            if not molrec['real'][iat]:
+                ghost_atoms.append(str(iat+1))
+
+        first_line = f"""{{{umap.get(units.lower())}}}"""
+        second_line = """geometry={"""
+        end_bracket = """}"""
+        smol = [first_line, second_line]
+        smol.extend(atoms)
+        smol.append(end_bracket)
+
```

Comment:
could you add the `set charge` and `mult` commands in here, too?

I'm favoring `to_string` returning all text that can be placed adjacent to the Cartesian lines, so I think chg/mult/dummy is in that.

Reply:
Yup that sounds good to me. The only thing that could potentially be problematic is we ever support specifying a separate xyz file to be read. But that's probably not worth worrying about right now.

Reply:
Agreed, I think xyz file is template only.